docs/site/index/cldr-spec/coverage-levels.md

```diff
@@ -72,12 +72,12 @@ Before submitting data above the Basic Level, the following must be in place:
 	- Must also include minimal pairs
 	- For more information, see [cldr\-spec/plural\-rules](/index/cldr-spec/plural-rules).
 2. Casing information (only where the language uses a cased scripts according to [ScriptMetadata.txt](https://github.com/unicode-org/cldr/blob/main/common/properties/scriptMetadata.txt))
-	- This will go into [common/casing](https://home.unicode.org/basic-info/projects/#!/repos/cldr/trunk/common/casing/)
+	- This will go into [common/casing]https://github.com/unicode-org/cldr/blob/main/common/casing/)
```

Comment:
should be [common/casing](https://github.com/unicode-org/cldr/blob/main/common/casing/)

Link is missing the (

Reply:
fixed, I think